com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/JavaDebugDelegateCommandHandler.java

```diff
@@ -30,6 +30,7 @@
     public static final String VALIDATE_LAUNCHCONFIG = "vscode.java.validateLaunchConfig";
     public static final String RESOLVE_MAINMETHOD = "vscode.java.resolveMainMethod";
     public static final String INFER_LAUNCH_COMMAND_LENGTH = "vscode.java.inferLaunchCommandLength";
+    public static final String DETECT_PREVIEW_FLAG = "vscode.java.detectEnablePreviewFeaturesFlag";
```

Comment:
I think this could be a more common API about fetch settings.

Reply:
The new API will return the java project settings map, and delay the check to the client side.

Reply:
This line is not necessary now.